doc-tool/src/dotty/tools/dottydoc/core/DocASTPhase.scala

```diff
@@ -108,9 +109,8 @@ class DocASTPhase extends Phase {
 
       /** objects, on the format "Object$" so drop the last letter */
       case o @ TypeDef(n, rhs) if o.symbol.is(Flags.Module) =>
-        val name = o.name.show
         //TODO: should not `collectMember` from `rhs` - instead: get from symbol, will get inherited members as well
-        ObjectImpl(o.symbol, annotations(o.symbol), name.dropRight(1), collectMembers(rhs),  flags(o), path(o.symbol).init :+ name, superTypes(o))
+        ObjectImpl(o.symbol, annotations(o.symbol), o.name.stripModuleClassSuffix.show, collectMembers(rhs),  flags(o), path(o.symbol), superTypes(o))
```

Comment:
I would do `o.name.stripModuleClassSuffix.toString`. `show` might add colors

Reply:
I don't think show adds color on names, but if we change this then there's a ton of places where this is used in dottydoc that need to be changed. I'll leave that to whoever wants to maintain dottydoc :).